service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersAboutToSubmitControllerTest.java

```diff
@@ -118,6 +136,46 @@ void shouldBuildNewOrderObject() {
         );
     }
 
+    @Test
+    void shouldBuildNewEPOObject() {
+        final ManageOrdersEventData manageOrdersEventData = buildManageOrdersEventData();
+        CaseData caseData = buildCaseData().toBuilder().manageOrdersEventData(manageOrdersEventData).build();
+
+        CaseData responseCaseData = extractCaseData(postAboutToSubmitEvent(caseData));
+
+        assertThat(responseCaseData.getOrderCollection()).containsOnly(
+            element(ELEMENT_ID, GeneratedOrder.builder()
+                .orderType("C23_EMERGENCY_PROTECTION_ORDER")
+                .title("C23 - Emergency protection order")
+                .children(CHILDREN)
+                .judgeAndLegalAdvisor(JudgeAndLegalAdvisor.builder()
+                    .judgeTitle(HIS_HONOUR_JUDGE)
+                    .judgeLastName("Dredd")
+                    .build())
+                .dateTimeIssued(now())
+                .approvalDateTime(manageOrdersEventData.getManageOrdersApprovalDateTime())
+                .childrenDescription("first1 last1, first2 last2")
+                .document(DOCUMENT_PDF_REFERENCE)
+                .unsealedDocumentCopy(DOCUMENT_WORD_REFERENCE)
+                .build()));
+    }
+
+    private ManageOrdersEventData buildManageOrdersEventData() {
+        return ManageOrdersEventData.builder()
+            .manageOrdersApprovalDateTime(now().minusDays(8))
+            .manageOrdersEndDateTime(now().minusDays(1))
+            .manageOrdersType(C23_EMERGENCY_PROTECTION_ORDER)
+            .manageOrdersEpoType(EPOType.PREVENT_REMOVAL)
+            .manageOrdersEpoRemovalAddress(Address.builder().addressLine1("address1").postcode("postcode").build())
+            .manageOrdersChildrenDescription("first1 last1, first2 last2")
+            .manageOrdersFurtherDirections("test directions")
+            .manageOrdersExclusionRequirement("Yes")
+            .manageOrdersWhoIsExcluded("John")
+            .manageOrdersExclusionStartDate(dateNow().plusDays(2))
+            .manageOrdersPowerOfArrest(DOCUMENT_PDF_REFERENCE)
+            .build();
+    }
+
```

Comment:
Would be useful to add your fields to the below method, so that we can check that they are removed at the end of the flow